Springboot/src/main/java/com/centralstudenthub/error/ConflictException.java

```diff
@@ -0,0 +1,7 @@
+package com.centralstudenthub.error;
+
+public class ConflictException extends Error {
+    public ConflictException(String message) {
```

Comment:
This class name is too vague
What does Conflict mean ?

Reply:
I implemented three types of exceptions.

- **Not Found Exception**: if I am trying to get record not in the database.
- **Already Exists Exception**: if I am trying to put record already in the database.
- **Conflict Exception**: if I am trying to put a record can't be in the database. Like assigning a course as a prerequisite for itself.

Reply:
Maybe change name to DatabaseLogicalConstraintException ?
The name Conflict can carry many meanings

Reply:
Good suggestion ✌️